### DISCUSSION TITLE
Bottom and Top Sidebars?

### DISCUSSION THREAD

**Discussion by Osteribo** [rstudio/bslib#1116] [Ideas]
I have a few use cases where a bottom and top sidebar would be excellent. Any ideas on how this could be implemented?

**Comment by udurraniAtPresage** [comment]
Something like this? https://rstudio.github.io/bslib/articles/sidebars/index.html#hello-sidebar

**Comment by gadenbuie** [comment]
I'd recommend looking into using an accordion for this https://rstudio.github.io/bslib/reference/accordion.html
